en.po

```diff
@@ -185,7 +185,7 @@ msgid "supervisor"
 msgstr "Supervisor"
 
 msgid "supervisorDescription"
-msgstr "Who approves your Expense Reports?"
+msgstr "Who in [Current entity] approves your Expense Reports?  If you're not sure or a foreigner, pick the National Operations Leader or consult with National Finance Team."
```

Comment:
I think `[Current entity]` is supposed to be replaced with the actual name of the user's entity. In the HR app, it is possible to do this with sprintf arguments.

```
msgstr "Who in %s approves your Expense Reports?  If you're not sure or a foreigner, pick the National Operations Leader or consult with National Finance Team."
```

Then, in the HR app code (I guess supervisorAutocomplete.f7.jsx?), you can pass in the entity name as an argument. Arguments must be passed in as an array. So something like:
```js
__("supervisorDesciption", [entityName])
```

Not too sure what is the best way to pass the entity name to SupervisorAutocomplete. Maybe `props` or `$store`?

Reply:
Thanks! I have fixed it.